### ISSUE TITLE
Make all the header tags conditional during HTML generation

### ISSUE THREAD

**Issue by warfare-plugins** [warfare-plugins/social-warfare#354]
```
		if( isset( $info[ 'meta_tag_values'][ 'og_image_height' ] ) && !empty( $info['meta_tag_values']['og_image_height'] ) ):
			$info['html_output'] .= PHP_EOL . '<meta property="og:image:height" content="'. trim( $info['meta_tag_values']['og_image_height'] ).'" />';
		endif;
```

Make each individual meta tag value check to make sure that value exists to prevent outputting any meta tags with blank values. Each tag should have an independent check specifically looking for the value of that specific value just as in the example above. These can be found in the swp_open_graph_html() function in social-warfare-pro/functions/header-meta-tags.php on line 243

**Comment by scott-wiser** [comment]
I believe we made a commit for this.  I'll see if I can find it.

**Comment by scott-wiser** [comment]
Ok I've modified each item in swp_open_graph_html and swp_twitter_card_html.  Each meta tag value will be checked if it's set and stopped if it's not.  They will also be checked for empty values.

I have tested the system on a local development environment, and did not note any meta tags with empty content, and each appeared to print fine.

I'm going to commit this.

For further testing, I'll keep an eye on this when using different data sets in what loads in the meta tags.

I also didn't trace the data sources to determine if they need to be encoded or sanitized, and I'll keep an eye open.
